freeipa.spec.in

```diff
@@ -148,7 +148,8 @@ BuildRequires:  pki-base-python2
 BuildRequires:  python-pytest-multihost
 BuildRequires:  python-pytest-sourceorder
 BuildRequires:  python-jwcrypto
-BuildRequires:  python-custodia
+# 0.3: sd-notify and ipaserver.secrets.service
```

Comment:
Nitpick: Please specify what new API is provided (`custodia.server`) rather than what requires it (`ipaserver.secrets.service`). Also I don't think sd-notify is a build dependency.